core/src/main/java/io/grpc/LoadBalancer.java

```diff
@@ -477,9 +478,35 @@ public void updateOobChannelAddresses(ManagedChannel channel, EquivalentAddressG
      * called again and a new picker replaces the old one.  If {@code updatePicker()} has never been
      * called, the channel will buffer all RPCs until a picker is provided.
      */
+    // TODO(zdapeng): add '@deprecated Please migrate ALL usages to {@link #updateBalancingState}'
```

Comment:
Please add a comment to say that using this method implies that this load balancer doesn't support channel state, and the application will get exception when trying to get the channel state.